page/jquery-mobile/theme-roller.md

```diff
@@ -5,25 +5,25 @@
 
 ## Theming Overview
 
-jQuery Mobile has a robust theme framework that supports up to 26 sets of toolbar, content, and button colors, called a "swatch". The framework comes with five defined themes (swatches "a" to "e") which can be used readily, removed, or overwritten.
+jQuery Mobile has a robust theme framework that supports up to 26 sets of toolbar, content, and button colors, called a "swatch". The framework comes with two defined themes A and B (light and dark) which can be used readily, removed, or overwritten.
 
 ### Default Theme Swatch Mapping for Components
 
-If no theme swatch letter is set at all, the framework uses the "a" swatch (black in the default theme) for headers and footers and the "c" swatch (light gray in the default theme) for the page content to maximize contrast between the both.
+If no theme swatch letter is set at all, the framework uses the "a" swatch (black in the default theme) for headers and footers.
 
-All items in containers inherit the swatch from their parent. Exceptions to this rule are the listdivider in listviews, the header of nested list pages, and the button of split button lists, which all default to "b" (blue in the default theme). Count bubbles default to "c" (silver in the default theme).
+All items in containers inherit the swatch from their parent. Exceptions to this rule are the listdivider in listviews, the header of nested list pages, and the button of split button lists, which all default to "b" (blue in the default theme).
```

Comment:
Sorry for the mistakes, I am still learning about jQM. So, I have compared the demos of the [newer version](https://demos.jquerymobile.com/1.4.5/theme-default/) and the [older version](https://demos.jquerymobile.com/1.4.5/theme-classic/). As per the comparision, listdivider in listviews do inherit the the swatch from their parent, the default swatch being "a" where it is light gray. The same goes for buttons of split button list and nested list pages.

Is my inference appropriate?